api/src/main/java/javax/enterprise/inject/spi/BeanManager.java

```diff
@@ -229,6 +231,41 @@
     public void fireEvent(Object event, Annotation... qualifiers);
 
     /**
+     * Fire an asynchronous event and notify observers.
```

Comment:
Good idea. I remove the fireAsyncEvent and add a the method. It would deprecate beanManager#fireEvent as well.